packages/terra-action-header/src/ActionHeader.module.scss

```
@@ -14,7 +14,10 @@
     background-size: contain;
     display: inline-block;
     position: relative;
-    top: var(--terra-action-header-icon-top, 0.1429rem);
+    @media screen and (-ms-high-contrast: active), (-ms-high-contrast: none) {
+      top: var(--terra-action-header-icon-top, 0.1429rem);
```

Let's rename this variable to the following to denote its scoped to Microsoft vendor prefix
```suggestion
      top: var(--terra-action-header-icon-ms-top, 0.1429rem);
```

---

But renaming it would be a breaking change.

---

Talked to Brett offline...if we proactively update this variable in the other themes, then it wouldn't break them and should be ok.

---

Resolved in 2ff8a50